sendrecv/gst/webrtc-sendrecv.c

```diff
@@ -103,26 +103,44 @@ handle_media_stream (GstPad * pad, GstElement * pipe, const char * convert_name,
     const char * sink_name)
 {
   GstPad *qpad;
-  GstElement *q, *conv, *sink;
+  GstElement *q, *conv, *resample = NULL, *sink;
```

Comment:
Why do you initialize `resample` here? This requires C99 and will break with C89 compilers.